src/main/java/uk/org/tombolo/core/utils/FixedValueUtils.java

```diff
@@ -51,4 +51,35 @@ public static int save(List<FixedValue> fixedValues){
             return saved;
         });
     }
+
+    /*
+	Save and update requries to check in the database whether the entry exists or not,
+	if exists it updates else adds, but that increase overhead and compute time.
+	Using this method will only keep the old value and discard the new one, in case of 
+	duplicate records.
+	Fix Me: Need to find a better way to address it
```

Comment:
Please, add a FIXME tag here so we can get it easily from the IDE and add an issue for it.

Reply:
done

Reply:
#530 is created